psamm_import/importer.py

```diff
@@ -123,6 +123,33 @@ def _decimal_representer(dumper, data):
     return dumper.represent_scalar('tag:yaml.org,2002:float', value)
 
 
+def default_compartment(model):
+    """If some compounds have no compartment, default compartment is made.
+
+    Returns what the default compartment should be set to.
+    """
+    default_compartment = 'c'
+    default_key = set()
+    suffix = 0
+    for reaction_id, reaction in iteritems(model.reactions):
+        if 'equation' not in reaction.properties:
+            continue
+
+        equation = reaction.properties['equation']
+        for compound, _ in equation.compounds:
+            default_key.add(compound.compartment)
+
+    if None in default_key and default_compartment in default_key:
+        suffix += 1
+        while True:
+            new_key = '{}_{}'.format(default_compartment, suffix)
+            if new_key not in default_key:
+                default_compartment = new_key
+                break
+            suffix += 1
+    return default_compartment
```

Comment:
I think it would be a good idea to produce a warning here if `None` was found in `default_key` i.e. the warning should only be printed if the model has any compounds without compartment. It should also show the name of the default compartment that was generated.